app/src/main/java/org/thoughtcrime/securesms/calls/WebRtcCallActivity.kt

```diff
@@ -429,13 +413,41 @@ class WebRtcCallActivity : PassphraseRequiredActionBarActivity() {
                         binding.floatingRenderer.isVisible = false
                     }
 
+                    // the floating video inset (empty or not) should be shown
+                    // the moment we have either of the video streams
+                    val showFloatingContainer = state.userVideoEnabled || state.remoteVideoEnabled
+                    binding.floatingRendererContainer.isVisible = showFloatingContainer
+                    binding.swapViewIcon.isVisible = showFloatingContainer
+
+                    // make sure to default to the contact's avatar if the floating container is not visible
+                    if(!showFloatingContainer) showAvatar(false)
```

Comment:
if <space> (condition)

Reply:
The avatar needs to be shown, and shown with the contact's avatar, only when the floating container is hidden. Not sure this can be simplified?

Reply:
There's a space after `if`

Reply:
```suggestion
                    if (!showFloatingContainer) showAvatar(false)
```

Reply:
Ah... I didn't realise we were going with this convention... 
There are a lot of  ifs without spaces in the code though.